src/java/test/org/apache/zookeeper/server/quorum/auth/QuorumDigestAuthTest.java

```diff
@@ -87,7 +87,7 @@ public void testValidCredentials() throws Exception {
         authConfigs.put(QuorumAuth.QUORUM_SERVER_SASL_AUTH_REQUIRED, "true");
         authConfigs.put(QuorumAuth.QUORUM_LEARNER_SASL_AUTH_REQUIRED, "true");
 
-        String connectStr = startQuorum(3, authConfigs, 3);
+        String connectStr = startQuorum(3, authConfigs, 3, false);
```

Comment:
This test is not depending on `KerberosSecurityTestcase ` and not required delay. My above comment is answering this question, kindly refer that.